### DISCUSSION TITLE
Differential peak analysis based on batch corrected cluster ID from Harmony

### DISCUSSION THREAD

**Discussion by jinzhuangdou** [GreenleafLab/ArchR#964] [Questions / Documentation]
Hi ArchR team,

I merged multiple samples and performed integration analysis. Cells are still clusters by patient to the most part based on standard LSI. After harmony batch correction, the results seem reasonable. I know the batch-correction is implemented on the low-dimensional space.  When I perform downstream differential peak analysis, are the peaks batch-corrected or still derived from original profiles? In scRNA-seq data of Seurat, there is an imputed RNA after integration. Does ArchR use the same way for peak files? Many thanks.

**Comment by rcorces** [comment]
Usage questions belong in the Discussions forum. Issues are only for bug or error reports. This is stated clearly in the issue template - 
![image](https://user-images.githubusercontent.com/10343046/128543134-12cf6203-e9aa-4692-8072-e91f4115a525.png)

I'm converting your issue to a Discussion.

**Reply by jinzhuangdou** [reply]
Sorry. I did not notice that.

**Comment by rcorces** [comment]
To answer your question, Harmony batch correction attempts to position similar cells together in lower dimensional space. Clusters are then identified in this Harmony-corrected space which forces cells to be called as part of the same cluster that would not have been part of the same cluster prior to Harmony. When you perform differential analyses on these clusters downstream, those differentials are derived from the un-corrected data which means that any features that are heavily batch confounded will effectively be ignored/not significant.

**Reply by jinzhuangdou** [reply]
I see. Thanks for you explanation. Another question is ArchR using tile matrix profile to perform dimension reduction. I found  cells are still clusters by samples to the most part based on standard LSI of ArchR. This still happens even I removed the first dimension of LSI which is highly correlated with fragment size.  But when I using standard LSI in Signac, the integration seems reasonable (technical replicates are grouped together). Do you have any thoughts on this inconsistence? Many thanks.

**Reply by rcorces** [reply]
Many parameters affect LSI and the implementations in ArchR and Signac are very different. I have not followed the development of Signac but it used to be the case that a peak matrix (called at the bulk level) was imported and used for dimensionality reduction. This uses a much smaller feature set and it is not surprising if it groups cells together.
